docs/licenses.md

```diff
@@ -0,0 +1,8 @@
+### Dependences compatibility by license
+
+| Name | License | Compatibility |
+|------|---------|---------------|
+| OpenBlas | BDS-3 | Yes |
+| Blaze | BSD-3 | Yes |
+| libjpeg-turbo | [It's complicated](https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/LICENSE.md)| Yes(?) |
+| CImg | CeCILL(-C) | Yes(?) |
```

Comment:
Because CeCILL is specific, add link to the text (looks like it is required by the license)

Reply:
CImg has two licenses CeCILL and CeCILL-C
Since we use static link it's illegal to use first one as I understand

Reply:
okay